src/tests/mam4_wet_deposition_unit_tests.cpp

```diff
@@ -0,0 +1,170 @@
+// mam4xx: Copyright (c) 2022,
+// Battelle Memorial Institute and
+// National Technology & Engineering Solutions of Sandia, LLC (NTESS)
+// SPDX-License-Identifier: BSD-3-Clause
+
+#include "testing.hpp"
+#include <mam4xx/mam4.hpp>
+
+#include <haero/constants.hpp>
+
+#include <ekat/ekat_type_traits.hpp>
+#include <ekat/logging/ekat_logger.hpp>
+#include <ekat/mpi/ekat_comm.hpp>
+
+#include <catch2/catch.hpp>
+#include <cmath>
+#include <iomanip>
+#include <iostream>
+#include <limits>
+#include <memory>
+
+using namespace haero;
+using namespace mam4;
+
+TEST_CASE("test_constructor", "mam4_wet_deposition_process") {
+  ekat::Comm comm;
+  ekat::logger::Logger<> logger("wet deposition constructor test",
+                                ekat::logger::LogLevel::debug, comm);
+  mam4::AeroConfig mam4_config;
+  mam4::WetDepositionProcess::ProcessConfig process_config;
+  mam4::WetDepositionProcess process(mam4_config, process_config);
+  REQUIRE(process.name() == "MAM4 Wet Deposition");
+  REQUIRE(process.aero_config() == mam4_config);
+}
+
+TEST_CASE("test_local_precip_production", "mam4_wet_deposition_process") {
+  ekat::Comm comm;
+  // TODO - figure out how to print this logging information...
```

Comment:
I don't know where this sort of thing is documented. I even noticed other tests were using blanket cout statements to print instead which was odd...

Reply:
Everyone has their own testing philosophy, so I'm not surprised that people are just doing Whatever. As you can see, our small, deadline-driven porting team leaves quite a few gaps in documentation and testing polish relative to larger and more established projects. :-)

What would be less odd to you? Do you think we should use `spdlog` for reporting messages in tests?

Reply:
ekat logging is fine, I just can't remember/not sure where it is documented how to print out the logging commands that I am putting in using that logger.

Reply:
I don't think we have docs for that, so if you want to make some stuff up, feel free!